src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/Exchanges.kt

```diff
@@ -0,0 +1,52 @@
+package io.polygon.kotlin.sdk.rest.reference
+
+import com.thinkinglogic.builder.annotation.Builder
+import io.ktor.http.*
+import io.polygon.kotlin.sdk.rest.PolygonRestOption
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/** See [PolygonReferenceClient.getExchangesBlocking] */
+@SafeVarargs
+suspend fun PolygonReferenceClient.getExchanges(params: ExchangesParameters, vararg opts: PolygonRestOption): ExchangesResponse =
+    polygonClient.fetchResult({
+        path("v3", "reference", "exchanges")
+
+        params.assetClass?.let { parameters["asset_class"] = it }
+        params.locale?.let { parameters["locale"] = it }
+    }, *opts)
+
+@Builder
+data class ExchangesParameters(
+
+    /**
+     * Filter by asset class.
+     */
+    val assetClass: String? = null,
+
+    /**
+     * Filter by locale.
+     */
+    val locale: String? = null,
+)
+
+@Serializable
+data class ExchangesResponse(
```

Comment:
It looks like the response also include a `count` attribute

Reply:
ah good catch 👍